powersimdata/input/converter/pypsa_to_grid.py

```diff
@@ -83,54 +81,50 @@ def _get_storage_storagedata(n, storage_type):
     return storage_storagedata
 
 
-def _get_storage_gencost(n, storage_type):
+def _get_storage_gencost(df, storage_type):
     """Get storage data from PyPSA for data frame "gencost" in PSD's storage
     dict.
 
-    :param pypsa.Network n: PyPSA network to read in.
+    :param pandas.DataFrame df: PyPSA component dataframe.
     :param str storage_type: key for PyPSA storage type.
     :return: (*pandas.DataFrame*) -- data frame with storage data.
     """
-    if storage_type == "storage_units":
-        df_gencost = n.storage_units
-    elif storage_type == "stores":
-        df_gencost = n.stores
-    else:
+    if storage_type not in ["storage_units", "stores"]:
```

Comment:
I think we can remove the warning at least from this function, since `storage_type` isn't used anymore.

Reply:
yes